documentation/asciidoc/topics/proc_uninstalling.adoc

```diff
@@ -0,0 +1,31 @@
+[id='uninstalling-helm-chart_{context}']
+= Uninstalling the Helm chart
+
+[role="_abstract"]
+Uninstall the {brandname} chart to remove pods and other deployment artifacts.
+
+[NOTE]
+====
+This procedure shows you how to uninstall the {brandname} chart and generated with the `helm` and `oc` clients.
+You can also uninstall the Helm chart through the {osweb}.
+====
+
+.Prerequisites
+
+* Deploy the {brandname} chart.
+
+.Procedure
+
+. Use the `helm` client to remove the {brandname} chart, for example:
++
+[source,bash,options="nowrap",subs=attributes+]
+----
+$ helm uninstall infinispan
+----
++
+. Use the `oc` client to remove the generated secret and any other deployment artifacts.
```

Comment:
It should just be the generated secret that needs removing. All other artifacts are automatically removed on delete/update/rollback, but the generated secrets can't be as they need to stay the same during update/rollback so that the user doesn't have to update their client setting.